services/dumper.js

```diff
@@ -180,15 +183,13 @@ function Dumper(config) {
   }
 
   function writeRoute(modelName) {
-    const modelNameDasherized = _.kebabCase(modelName);
```

Comment:
Is it a cleaning of the existing code?
If so it should not be present in this PR to make it as easy as possible to read.

I am a bit picky here, but I think we should do that to make the best collaboration/review possible.
Every tiny details could be noise for the reviewer.

(See Pull Requests - Code of Conduct: https://forest.slite.com/app/channels/-KdBU14ToWBAerCk_HGS/notes/QFFoNQ1pOg#68229e06)

Reply:
Sorry about that, you are right ! it's difficult to loose bad practice 🤣